src/cloudant/index.py

```diff
@@ -262,3 +262,91 @@ def delete(self):
         """
         msg = 'Deleting the \"special\" index is not allowed.'
         raise CloudantException(msg)
+
+class SearchIndex(dict):
+    """
+    Encapsulates a SearchIndex as a dictionary based object, exposing the
+    search index function and analyzer as attributes.
+    """
+    def __init__(
+            self,
+            ddoc,
+            search_index_name=None,
+            search_index=None
+    ):
+        super(SearchIndex, self).__init__()
+        self.design_doc = ddoc
+        if search_index_name is not None:
+            self._search_index_name = search_index_name
+        if search_index is not None:
+            self['index'] = codify(search_index)
+        self['analyzer'] = codify('standard')
+
+    @property
+    def analyzer(self):
+        """
+        Provides an search index analyzer property accessor and setter.
+
+        For example:
+
+        .. code-block:: python
+
+            # Set the SearchIndex index property
+            search.analyzer = 'standard'
+
+            print search.analyzer
```

Comment:
Removed code block and added link to analyzer doc: https://github.com/cloudant/python-cloudant/pull/169/files#diff-eeec48d80ce666f16517361d09cf63f1R304